backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/LocalOnlyIssue.java

```diff
@@ -84,4 +86,7 @@ public void resolve(IssueStatus newStatus) {
     resolution = new LocalOnlyIssueResolution(newStatus, Instant.now(), null);
   }
 
+  public String getRuleDescriptionContextKey() {
```

Comment:
Not sure I understand why you had to modify this class and the corresponding storage. I don't think it's needed

Reply:
This was probably a leftover from the time when I tried to fetch issues from `LocalOnlyIssueStore`